portfolio/src/main/webapp/index.html

```diff
@@ -119,7 +119,7 @@ <h2 id="outro">...and here we are today.</h2>
                         </p>
                     </div>
                 </div>
-                <div class="msg"> 
+                <div class="msg">
```

Comment:
There seem to be a bunch of changes in this PR that are not related to meeting scheduling. I would suggest splitting moving the non-meeting related changes to a separate PR.

Reply:
Actually I think this happened while I was rebasing (when deleting things, I deleted extra space, etc).